rules/windows/process_creation/win_process_dump_rdrleakdiag.yml

```diff
@@ -0,0 +1,22 @@
+title: Process Dump via RdrLeakDiag.exe
+id: EDADB1E5-5919-4E4C-8462-A9E643B02C4B 
```

Comment:
Hi id is a uuidV4 lowercase

Reply:
Hi, ok, I obtained uuidV4 uppercase on my macOs ^^. The change has been made.